docs/compendium/substitute.md

```diff
@@ -25,7 +36,90 @@ introduction...
 ### Error responses
 
 ```json
+401 Unauthorized
+
+{"error":"not authenticated or not allowed"}
+```
+
+```json
+404 Not Found
+
+{"error":"compendia ids not found"}
+```
+
+### Request
+
+`POST /api/v1/substitution`
+
+input of request-body for substitution <br/>
+( where `checked` will be a list of files choosen for substitution )
+```json
+{
+  "user": "user_id",
+  "ercs": [
```

Comment:
Do you expect to have a list longer than three? I suggest to stick to specific "base" and "overlay". In your draft, how can I identify the overlay?

```json
{
  "user": "user_id",
  "base": {
      "id": "adf123",
      "checked": []
    },
    {
      "id": "1234adb",
      "checked": []
    }
  ]
}
```